src/components/input/input.tsx

```diff
@@ -121,6 +121,11 @@ export class Input implements LabelableComponent, FormComponent, InteractiveComp
   /** BCP 47 language tag for desired language and country format */
   @Prop() locale?: string = document.documentElement.lang || "en";
 
+  /**
+   * standard UniCode numeral system tag for localization
+   */
+  @Prop() numberingSystem?: string = new Intl.NumberFormat().resolvedOptions().numberingSystem;
```

Comment:
If this is an optional property, it shouldn't have a default value set. If it has a default value, then the `?` is ignored since a value is already supplied. I think the above `locale` is incorrect as well.

Maybe for these we shouldn't set a default value on the component? We can compute the value internally if we need to but not set it on the component prop.

What do you think?

Reply:
Does `new Intl.NumberFormat().resolvedOptions().numberingSystem` need to be called on every instance of the component?

Reply:
It makes sense to remove the default value as the i18n team confirmed that CLDR defines a default numbering system for each locale . And it doesn't have to be called on every instance unless there is a change in locale / numbering System prop.